File: infra/lib/dynamodb.ts
import * as dynamodb from '@aws-cdk/aws-dynamodb';
import { Stack } from '@aws-cdk/core';

export default function(stack: Stack) {
  const doctors = new dynamodb.Table(stack, 'doctors', {
    partitionKey: {name: 'id', type: dynamodb.AttributeType.STRING },
    billingMode: dynamodb.BillingMode.PROVISIONED
  });

  const patients = new dynamodb.Table(stack, 'patients', {
    partitionKey: {name: 'id', type: dynamodb.AttributeType.STRING },
    billingMode: dynamodb.BillingMode.PROVISIONED
  });

  return { doctors, patients };
}

File: infrastructure/infrastructure/dynamodb.py
from aws_cdk import core, aws_dynamodb

class DynamoDB:
    def __init__(self, stack: core.Stack):
        self.doctors = aws_dynamodb.Table(stack, "doctors",
            partition_key=aws_dynamodb.Attribute(
            name="id",
            type=aws_dynamodb.AttributeType.STRING))
